hypershift-operator/controllers/nodepool/manifests.go

```diff
@@ -37,6 +40,58 @@ func machineHealthCheck(nodePool *hyperv1.NodePool, controlPlaneNamespace string
 	}
 }
 
+func KubevirtMachineTemplate(infraName string, hostedCluster *hyperv1.HostedCluster, nodePool *hyperv1.NodePool, controlPlaneNamespace string) (*capikubevirt.KubevirtMachineTemplate, string) {
```

Comment:
Let's put this into its own kubevirt.go file in this package.

Reply:
Done